app/Rules/ValidateSafeSvg.php

```
@@ -0,0 +1,50 @@
+
+declare(strict_types=1);
+
+namespace App\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Http\UploadedFile;
+
+class ValidateSafeSvg implements ValidationRule
+{
+    public function validate(string $attribute, mixed $value, Closure $fail): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)